Ginger/Ginger/ALM/ALMConnectionPage.xaml.cs

```diff
@@ -221,7 +222,7 @@ private bool GetProjectsDetails()
             {
                 isProjectMappingCorrect = false;
             }
-            SetControls();
+            //SetControls();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6008688)